webapp/channels/src/i18n/en.json

```diff
@@ -1377,6 +1359,8 @@
   "admin.license.modal.upload": "Upload",
   "admin.license.modal.uploading": "Uploading",
   "admin.license.no-file-selected": "No file selected",
+  "admin.license.notices.enterprise": "This software is offered under a commercial license. See ENTERPRISE-EDITION-LICENSE.txt in your root install directory for details. See NOTICE.txt for information about open source software used in this system.",
```

Comment:
I don't see why this wouldn't be translated. The file names or contents won't be translated for sure, but the rest of the text doesn't sound like it shouldn't be translated.

Reply:
I think the text of the license legally can't be translated since that could change the meaning of the contract. I don't think it's a huge deal here if this is just used for search indexing, but I'm pretty sure that's why these lines were hardcoded previously. I feel like it'd be safer to continue not translating these lines

Reply:
Thanks for reverting the rest of this, but these strings are still causing check-i18n to fail since they're now unused